src/cloud/lib/mappers/topbarTree.ts

```diff
@@ -57,6 +57,7 @@ export function mapTopbarTree(
   )
 
   folders.forEach((folder) => {
+    if (folder.pathname == '/') return
```

Comment:
Seems like a mistake, changing this in cloud mapper, I'll revert this.
I suppose its not a mistake in rebasing